archetypes/business-service-spring-boot2-starter/src/main/resources/archetype-resources/pom.xml

```diff
@@ -28,7 +28,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- this version is used for archetype, do not need to change when release a new version -->
     <java-chassis.version>2.0.0</java-chassis.version>
-    <springboot.version>2.1.2.RELEASE</springboot.version>
+    <springboot.version>2.5.6</springboot.version>
```

Comment:
Should change java-chassis.verion and springboot.version together. java chassis 2.0.0 may not work with spring boot 2.5.6.